src/http_server/api/v1/metrics.h

```diff
@@ -26,4 +26,11 @@
 
 int api_v1_metrics(struct flb_hs *hs);
 
+struct metric_help {
+    char *text;
+    int len;
+};
+
+void metrics_help_txt(char *metric_name, struct metric_help **mh);
+
```

Comment:
Why not use an SDS in place of metric help? 

SDS strings store a `char *` and a length; see: https://github.com/fluent/fluent-bit/blob/master/include/fluent-bit/flb_sds.h#L51

Reply:
@PettitWesley, thanks for your advise, you have the right questions. ;-) I think i didn't see the forrest for the trees. 
I will use the sds structures...